libnopegl/src/backends/gl/pipeline_gl.c

```diff
@@ -368,7 +368,7 @@ static void set_vertex_attribs(const struct pipeline *s, struct glcontext *gl)
         const struct buffer_gl *buffer_gl = (const struct buffer_gl *)buffer;
         const GLuint location = attribute_binding->desc.location;
         const GLuint size = ngli_format_get_nb_comp(attribute_binding->desc.format);
-        const GLint stride = attribute_binding->desc.stride;
+        const GLsizei stride = (GLsizei)attribute_binding->desc.stride;
```

Comment:
` pipeline: set buffer+attributes stride and offset to size_t ` -> ` pipeline: set buffer/attribute stride and offset to size_t

Reply:
Done

Reply:
Sorry, I forgot to mention `set` -> `change`.